drivers/frequency/adf4377/adf4377.c

```diff
@@ -31,11 +32,73 @@
  * EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *******************************************************************************/
 
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
 #include <malloc.h>
 #include "adf4377.h"
 #include "no_os_error.h"
 #include "no_os_delay.h"
 #include "no_os_alloc.h"
+#include "no_os_print_log.h"
+#include "no_os_util.h"
+
+/******************************************************************************/
```

Comment:
dont put these code delimiters, we removed them from main as well, delete them from all new files